test/jest/playwrightEnvironment.js

```diff
@@ -37,6 +38,8 @@ class PlaywrightEnvironment extends NodeEnvironment {
     testOptions.USES_HOOKS = process.env.PWCHANNEL === 'wire';
     testOptions.CHANNEL = !!process.env.PWCHANNEL;
     testOptions.HEADLESS = !!valueFromEnv('HEADLESS', true);
+    testOptions.OUTPUT_DIR = path.join(__dirname, '..', 'output-' + browserName);
+    fs.mkdirSync(testOptions.OUTPUT_DIR, { recursive: true });
```

Comment:
This does not work with the multiple workers. I have a global set up implemented in my bulk conversion: https://github.com/microsoft/playwright/pull/3002/files